docs/advanced/program-options.md

```diff
@@ -63,8 +63,8 @@ Option                      | Description
 `--testnet `                | Use the test network
 `--simnet`                  | Use the simulation test network
 `--nocheckpoints=`          | Disable built-in checkpoints. Don't do this unless you know what you're doing.
-`--dbtype=`                 | Database backend to use for the Block Chain (leveldb)
-`--profile=`                | Enable HTTP profiling on given port -- NOTE port must be between 1024 and 65536 (6060)
+`--dbtype=`                 | Database backend to use for the blockchain (leveldb)
```

Comment:
This is showing the output of `dcrd -h` so you should not be changing this.  It needs to match what the command does.